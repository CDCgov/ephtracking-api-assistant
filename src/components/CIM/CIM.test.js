import React from 'react';
import CIM from '.';
import { shallow } from 'enzyme';

describe('CIM Component', () => {
  it('renders correctly', () => {
    const handleSelect = event => console.log('handleSelect');
    const tree = shallow(<CIM handleSelect={handleSelect} />);
    expect(tree).toMatchSnapshot();
  });

});
