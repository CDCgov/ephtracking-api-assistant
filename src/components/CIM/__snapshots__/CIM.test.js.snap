// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CIM Component renders correctly 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <CIM
    handleSelect={[Function]}
/>,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "function",
    "props": Object {
      "children": Array [
        <SelectContentArea
          handleSelect={[Function]}
/>,
        <SelectIndicator
          contentAreaId={null}
          handleSelect={[Function]}
/>,
        <SelectMeasure
          handleSelect={[Function]}
          indicatorId={null}
/>,
      ],
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "class",
        "props": Object {
          "handleSelect": [Function],
        },
        "ref": null,
        "rendered": null,
        "type": [Function],
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "class",
        "props": Object {
          "contentAreaId": null,
          "handleSelect": [Function],
        },
        "ref": null,
        "rendered": null,
        "type": [Function],
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "class",
        "props": Object {
          "handleSelect": [Function],
          "indicatorId": null,
        },
        "ref": null,
        "rendered": null,
        "type": [Function],
      },
    ],
    "type": Symbol(react.fragment),
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "function",
      "props": Object {
        "children": Array [
          <SelectContentArea
            handleSelect={[Function]}
/>,
          <SelectIndicator
            contentAreaId={null}
            handleSelect={[Function]}
/>,
          <SelectMeasure
            handleSelect={[Function]}
            indicatorId={null}
/>,
        ],
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "class",
          "props": Object {
            "handleSelect": [Function],
          },
          "ref": null,
          "rendered": null,
          "type": [Function],
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "class",
          "props": Object {
            "contentAreaId": null,
            "handleSelect": [Function],
          },
          "ref": null,
          "rendered": null,
          "type": [Function],
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "class",
          "props": Object {
            "handleSelect": [Function],
            "indicatorId": null,
          },
          "ref": null,
          "rendered": null,
          "type": [Function],
        },
      ],
      "type": Symbol(react.fragment),
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
